app/views/dataset.scala.html

```diff
@@ -382,6 +382,24 @@ <h4>Statistics</h4>
     <script src="@routes.Assets.at("javascripts/deleteUtils.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/people.js")" type="text/javascript"></script>
     <script src="@routes.Assets.at("javascripts/creators.js")" type="text/javascript"></script>
+
+    <script type="application/ld+json">{"context": { "@@vocab": "https://schema.org/"},
+		 "identifier": "@dataset.id",
+		 "name": "@dataset.name",
+		 "author": @Html(dataset.author.to_jsonld()),
+		 "description": "@dataset.description",
+		 "dateCreated": "@dataset.created.format("MMM dd, yyyy")",
+		 "DigitalDocument": "@dataset.files.mkString(",")",
```

Comment:
can we use a link to the file page instead, also limit to first 10 or so (in case we have 100,000 files in a dataset).

Reply:
I will go with the number returned by dataset.files and will have to map then to page links

Reply:
Just need to check on this, tried to resolve other conversations, but it would not let me

Reply:
Been trying to find CLOWDER_URL internally so I can: files.map(f => clowder_host + "files/" + f)

Reply:
use that env var to override default, and set in Server global, to use for mapping; works well unless there is a preferred place for Server url